pex/resolve/lock_resolver.py

```diff
@@ -20,6 +20,7 @@
 from pex.pip.local_project import digest_local_project
```

Comment:
The get_pip import above this line is not used

Reply:
(My IDE pointed it out, that is not some Jedi-level code grokking on my part...)

Reply:
Yeah - I generally hire the same ninja. I run IDEA's Optimize Imports over the whole change then `tox -eformat`. Missed it this time!